benchexec/model.py

```diff
@@ -107,12 +107,25 @@ def load_task_definition_file(task_def_file):
             f"invalid format_version '{task_def.get('format_version')}'."
         )
 
-    if format_version != "2.0" and "options" in task_def:
+    if format_version not in ["2.0", "2.1"] and "options" in task_def:
         raise BenchExecException(
             f"Task-definition file {task_def_file} specifies invalid key 'options', "
             f"format_version needs to be at least 2.0 for this."
         )
 
+    if format_version != "2.1" and "additional_information" in task_def:
+        raise BenchExecException(
+            f"Task-definition file {task_def_file} specifies invalid key 'additional_information', "
+            f"format_version needs to be at least 2.1 for this."
+        )
+
+    if format_version == "2.1":
+        # Remove the additional_information key from the task definition
+        # since this should only be considered as a comment
+        # as described in its definition:
+        # https://gitlab.com/sosy-lab/benchmarking/task-definition-format
+        task_def.pop("additional_information", None)
+
```

Comment:
```suggestion
```
We can simply remove this. I don't see how having the key in memory could lead to accidental use of it in BenchExec. And it might be really confusing when one starts intentionally using it (it is not a comment, cf. my response on the MR).